Automattic/woocommerce-payments
# Show WCPay Subscription ID on admin edit subscription screen

Fixes #3196

#### Changes proposed in this Pull Request

This PR shows the WCPay Subscription ID when viewing the Edit Subscription page for WCPay subscriptions, for reference during support/troubleshooting.

<img width="372" alt="WCPay Subscription ID" src="https://user-images.githubusercontent.com/16066224/139066386-e2d3faf2-511e-4b7e-84ad-e265f24e6638.png">

#### Testing instructions

1. Enable WCPay Subscriptions.
2. As a customer, purchase a new subscription product.
3. As admin, go to WooCommerce > Subscriptions and click on the subscription number.
4. Verify that you can see the subscription ID (starts with `sub_`) under **Billing**.
5. In Stripe, visit `https://dashboard.stripe.com/test/connect/accounts/ACCOUNT_ID/subscriptions/SUBSCRIPTION_ID` where `ACCOUNT_ID` is your Stripe Connect Account ID and `SUBSCRIPTION_ID` is the ID from the above step. Verify that you see the correct subscription details.
6. As admin, access other order pages that are not WCPay Subscriptions (e.g. manual renewal subscriptions, orders that aren't subscriptions) and verify that you do not see a WCPay Subscription ID.

-------------------


**Post merge**
